src/lib/utils/breadcrumbUtils.tsx

```diff
@@ -0,0 +1,69 @@
+import { BreadcrumbItem } from '@/types/dataTypes/drupal/field_type'
```

Comment:
**SHOULD**
Can we rename this to just breadcrumbs.ts?
- Don't need `.tsx`
- Don't need `Utils` (it's redundant given the directory structure)

Reply:
Agreed, fixed